modules/ROOT/pages/dataweave-formats-flatfile.adoc

```diff
@@ -23,17 +26,108 @@ output application/flatfile schemaPath="src/main/resources/test-data/QBReqRsp.es
 payload
 ----
 
+// CONFIG PROPS GET APPENDED BELOW /////////////////////////////////
+
+
+[[properties]]
+== Configuration Properties
+
+DataWeave supports the following configuration properties for this format.
+
+
+[[reader_properties]]
+=== Reader Properties
+
+This format accepts properties that provide instructions for reading input data.
+
+
+[cols="1,1,1,3a", options="header"]
+|===
+|Parameter |Type |Default|Description
+|`enforceRequires`  |`Boolean`|`false`|Produces an error when set to `true` if a required value is missing.
+
+Valid values are `true` or `false`.
+|`missingValues`  |`String`|`null`|Specifies one of the following fill characters to represent missing values: 
```

Comment:
Pending
TODO: JIRA bc null is incorrect. There is no null value option.

It's "nulls for copybook schema, spaces otherwise" AND we almost certainly need to separate out defaults based on the type (copybook, flat file, fixed width).